### PR TITLE
chore: update readme for GA

### PR DESCRIPTION
**Pull request by ascorbic**
This PR updates the readme to reflect the changes for version 2 GA.

[Rendered version](https://github.com/netlify/netlify-plugin-gatsby/blob/mk/ga-readme/README.md)

### CODE REVIEW

**Review by tiffanosaurus — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -20,58 +18,42 @@ which is installed automatically for new Gatsby sites.
...
-- [Install version 2 beta of `@netlify/plugin-gatsby`](#install-the-netlify-build-plugin)
-  (the Netlify build plugin)
-- [Install version 4 beta of `gatsby-plugin-netlify`](#install-the-gatsby-plugin)
-  (the Gatsby plugin)
+1. Search for "Essential Gatsby" and install the plugin
+   [from the Netlify UI](https://docs.netlify.com/configure-builds/build-plugins/#ui-installation)
 
-### Install the Netlify build plugin
+2. Install the plugin as `@netlify/plugin-gatsby` using
+   [file-based plugin installation](https://docs.netlify.com/configure-builds/build-plugins/#file-based-installation).
```
Comment (KyleBlankRollins) [inline README.md:41]: [pebble] Same as the previous list item.

```suggestion
- [File-based plugin installation](https://docs.netlify.com/configure-builds/build-plugins/#file-based-installation). You can install the plugin as `@netlify/plugin-gatsby` in your `netlify.toml` file.
   
```

File: README.md [hunk 2]
```diff
@@ -20,58 +18,42 @@ which is installed automatically for new Gatsby sites.
...
+1. The Netlify build plugin, called "Essential Gatsby" or
+   `@netlify/plugin-gatsby`. This is installed automatically for all Gatsby
+   sites deployed to Netlify.
+2. The Gatsby plugin `gatsby-plugin-netlify`, which needs to be manually
+   installed.
 
-You're able to
-[remove the plugin](https://docs.netlify.com/configure-builds/build-plugins/#remove-a-plugin)
-at any time by visiting the **Plugins** tab for your site in the Netlify UI. -->
+### Installing the Netlify build plugin
```
Comment (KyleBlankRollins) [inline README.md:31]: [sand] This header uses "Installing" while the following header (line 43) uses "Install". They should use the same verb form. I advocate using "installing" for both, but you could make an argument either way.

**Review by KyleBlankRollins — DISMISSED** [review]
Looks great! Nice write up, @ascorbic. I have some suggestions, but nothing blocking. Mostly minor grammar/reading optimization things.

**Comment by ascorbic** [comment]
@KyleBlankRollins All great suggestions, thanks!

**Review by tiffanosaurus — APPROVED** [review]

**Pull request merged**